### ISSUE TITLE
Is there a reshaping of the gt density maps needed?

### ISSUE THREAD

**Issue by ciampluca** [jerpelhan/DAVE#15]
Hi, thank you for your work.

It seems that during the data loading, the gt density maps should be in the folder `gt_density_map_adaptive_512_512_object_VarV2`

However, this folder does not exist in the original dataset. I guess that they are the reshaped original density maps. Did you pad the density masks? What kind of padding did you use? Did you reshape density maps in some other ways?

Thank you

**Comment by jerpelhan** [comment]
Yes, that's correct! If you want to train the model from scratch, refer to [LOCA](https://github.com/djukicn/loca) for all the necessary steps to generate density maps. LOCA uses adaptively changed Gaussians based on the width and height of the exemplars in the training stage. However, for evaluation, everything should work properly with the original density maps. I haven't checked, but I'm fairly certain it should.

**Comment by ciampluca** [comment]
> Yes, that's correct! If you want to train the model from scratch, refer to [LOCA](https://github.com/djukicn/loca) for all the necessary steps to generate density maps. LOCA uses adaptively changed Gaussians based on the width and height of the exemplars in the training stage. However, for evaluation, everything should work properly with the original density maps. I haven't checked, but I'm fairly certain it should.

Thank you for your answer.
I think that it is needed not only for training, but also for other scripta such as fscd_0shot_clip.sh

Btw, I will follow the code of LOCA, thank you again.
